macroslib/src/python/python-include.rs

```diff
@@ -0,0 +1,29 @@
+
+#[macro_use] extern crate cpython;
```

Comment:
Unfortunately yes, it doesn't support 2018 edition. Also, it uses a lot of inner helper macros, so `#[macro_use]` is AFAIK the best method for now. I also think, that it is better to add it here and not rely on a user, because if (s)he misses it, errors messages will be quite cryptic.